lib/ci-stack.ts

```diff
@@ -54,18 +59,22 @@ export class CIStack extends Stack {
 
     const useSslParameter = `${props?.useSsl ?? this.node.tryGetContext('useSsl')}`;
     if (useSslParameter !== 'true' && useSslParameter !== 'false') {
-      throw new Error('useSsl parameter is required to be set as - true or false');
+      throw new Error('useSsl parameter is required to be set as - true or false. Please pass the value');
     }
 
     const useSsl = useSslParameter === 'true';
 
     const runWithOidcParameter = `${props?.runWithOidc ?? this.node.tryGetContext('runWithOidc')}`;
     if (runWithOidcParameter !== 'true' && runWithOidcParameter !== 'false') {
-      throw new Error('runWithOidc parameter is required to be set as - true or false');
+      throw new Error('runWithOidc parameter is required to be set as - true or false. Pass the damn value');
```

Comment:
Please change to `Please pass the value` if at all required

Reply:
Updated.